yt_dlp/extractor/tv5mondeplus.py

```diff
@@ -13,28 +15,33 @@ class TV5MondePlusIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?(?:tv5mondeplus|revoir\.tv5monde)\.com/toutes-les-videos/[^/]+/(?P<id>[^/?#]+)'
     _TESTS = [{
         # movie
-        'url': 'https://revoir.tv5monde.com/toutes-les-videos/cinema/ceux-qui-travaillent',
-        'md5': '32fa0cde16a4480d1251502a66856d5f',
+        'url': 'https://revoir.tv5monde.com/toutes-les-videos/cinema/les-novices',
```

Comment:
Don't replace tests. If these tests are dead, add `'skip': '[reason for skipping]'` to the test dict and add new tests